lib/help.js

```diff
@@ -11,7 +11,14 @@ var helpjson = {
 	'submit'			: 'To submit the code to JavaAutolab and fetch the results',
 	'help'				: 'Print help manual'
 };
+/**
+* Displays the list of autolab commands along with their functions.
+* @function help
+* @param {null}
+* @return {object} object
+*/
 module.exports = function() {
+
```

Comment:
see the relevant JSDoc option for specifying module exports.

BTW, the function does not return anything. It only prints to the console. But the function (object) itself would be available across all other places where lib/help.js is required.